vite.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
   publicDir: 'public',
   resolve: {
     alias: {
-      '@': '/src'
+      '@': join(__dirname, 'src')
```

Comment:
`dist/components/QuestDetailsWrapper/index.d.ts` was building the `@/types/quests` into `../../../../../../src/types/quests`

This now matches what we have in the desktop client and the `QuestWrapperContextValue` type works now

Reply:
niiiiice, this is the fix I was looking for but couldn't get